spec/features/admin_dashboard_spec.rb

```diff
@@ -1,17 +1,19 @@
 require "rails_helper"
+
 RSpec.feature "Submit application with minimal information" do
   before do
     page.driver.browser.current_session.basic_authorize("admin", "password")
   end
   scenario "The stats are accurate", javascript: true do
     10.times { create(:snap_application, signed_at: nil) }
-    1.times { create(:snap_application, :emailed, signed_at: nil) }
+    1.times { create(:snap_application, :emailed_client, signed_at: nil) }
     9.times { create(:snap_application, signed_at: 5.days.ago) }
     1.times { create(:snap_application, :faxed, signed_at: 5.days.ago) }
     visit admin_root_path
     expect(page).to have_content "21 Created"
     expect(page).to have_content "10 Signed (47.6%)"
-    expect(page).to have_content "1 Emailed (4.8%)"
+    expect(page).to have_content "1 Emailed Client (4.8%)"
+    expect(page).to have_content "0 Emailed Office (0.0%)"
```

Comment:
perhaps we should add an office email in the setup here so we can be sure the stats are working?